### PR TITLE
Template System documentation

### PR DESCRIPTION
**Pull request by ahilles107**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | no
| New feature?              | no
| BC breaks?                | yes
| Deprecations?             | no
| Tests pass?               | yes
| Changelog update required | no
| Fixed tickets             | comma separated list of tickets fixed by the PR
| License                   | AGPLv3

### CODE REVIEW

File: src/SWP/Bundle/TemplateEngineBundle/Service/ContainerService.php [hunk 1]
```diff
@@ -68,8 +72,13 @@ public function getContainer($name, array $parameters = [], $createIfNotExists =
         foreach ($containerWidgets as $containerWidget) {
             $widgetModel = $containerWidget->getWidget();
             $widgetClass = $widgetModel->getType();
-            $widgetHandler = new $widgetClass($widgetModel);
-            $widgetHandler->setContainer($this->serviceContainer);
+
+            if (is_a($widgetClass, '\SWP\Bundle\TemplateEngineBundle\Widget\TemplatingWidgetHandler', true)) {
```
Comment (takeit) [inline src/SWP/Bundle/TemplateEngineBundle/Service/ContainerService.php]: Do you need a function call here (`is_a`) ? what about just:

```php
if ($widgetClass instanceof TemplatingWidgetHandler::class) {
    // ..
}
```

File: src/SWP/Component/TemplatesSystem/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-==================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
+Templates System [![Build Status](https://travis-ci.org/SuperdeskWebPublisher/templates-system.svg?branch=master)](https://travis-ci.org/SuperdeskWebPublisher/templates-system) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/SuperdeskWebPublisher/templates-system/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/SuperdeskWebPublisher/templates-system/?branch=master) [![SensioLabsInsight](https://insight.sensiolabs.com/projects/4910c72c-f459-41b9-9724-b093dc15c525/mini.png)](https://insight.sensiolabs.com/projects/4910c72c-f459-41b9-9724-b093dc15c525)
```
Comment (takeit) [inline src/SWP/Component/TemplatesSystem/README.md]: `Templates System Component`

File: src/SWP/Component/TemplatesSystem/README.md [hunk 3]
```diff
@@ -14,20 +14,20 @@ Features
...
------------
+-------
 
 See the complete license [here](LICENSE.md).
 
 Contributors
--------
+------------
 
 This component is a Sourcefabric z.ú. and contributors initiative.
```
Comment (takeit) [inline src/SWP/Component/TemplatesSystem/README.md]: You could also change `component` to `project` as this is main read me file

File: src/SWP/Bundle/TemplateEngineBundle/Service/ContainerService.php (hunk 1 above)
Comment (ahilles107) [inline src/SWP/Bundle/TemplateEngineBundle/Service/ContainerService.php]: `$widgetClass` is a string.

**Pull request merged**